terraform/aws/dag

```diff
@@ -29,17 +27,6 @@ checkHosts() {
     echo -e "${red}HOSTS_FILE was not set${clr}"
     exit 1
   fi
-
-  if [ $CONFIRMED -eq 0 ]; then
```

Comment:
Please don't remove it

Reply:
Then I can add a parameter like `auto-approve` (I need to run it on CI)

Reply:
Yes, you can add something which will skip checks but checks must be there enabled by default.